cmd/kill.go

```diff
@@ -47,6 +62,12 @@ var killJobCmd = &cobra.Command{
 	Run:   killJob,
 }
 
+var killTasksCmd = &cobra.Command{
```

Comment:
please add more description how to use the command.

Reply:
Added description